modular_zubbers/code/modules/clothing/suits/armor.dm

```diff
@@ -55,3 +55,81 @@
 	name = "security jacket"
 	desc = "A blue jacket in blue Peacekeeper colors. It has hi-vis stripes all over it."
 	icon_state = "secjacket_blue"
+
+/obj/item/clothing/suit/armor/vest/intern // Bubberstation Edit
```

Comment:
You don't need the edit comment in modular files.

Reply:
> You don't need the edit comment in modular files.

Probably forgot to remove that-

Reply:
> You don't need the edit comment in modular files.

Fixed.